code/API_definitions/location-verification.yaml

```diff
@@ -564,50 +566,65 @@ components:
       content:
         application/json:
           schema:
-            $ref: "#/components/schemas/ErrorInfo"
+            allOf:
+              - $ref: "#/components/schemas/ErrorInfo"
+              - type: object
+                properties:
+                  status:
+                    enum:
+                      - 422
+                  code:
+                    enum:
+                      - AREA_NOT_COVERED
+                      - IDENTIFIER_MISMATCH
+                      - SERVICE_NOT_APPLICABLE
+                      - MISSING_IDENTIFIER
+                      - UNSUPPORTED_IDENTIFIER
+                      - UNNECESSARY_IDENTIFIER
+                      - LOCATION_VERIFICATION.UNABLE_TO_FULFILL_MAX_AGE
           examples:
-            LOCATION_VERIFICATION_422_UNABLE_TO_FULFILL_MAX_AGE:
-              summary: Unable to fulfill maxAge
-              description: The system is not able to provide the fresh location required by the client
+            GENERIC_422_AREA_NOT_COVERED:
               value:
                 status: 422
-                code: LOCATION_VERIFICATION.UNABLE_TO_FULFILL_MAX_AGE
-                message: "Unable to provide expected freshness for location"
-            GENERIC_422_UNPROCESSABLE_ENTITY:
-              summary: Unprocessable entity
-              description: The request was well-formed but was unable to be processed due to semantic errors or not applicable values. This is the generic error code for 422 responses.
+                code: AREA_NOT_COVERED
```

Comment:
We may need different codes for both cases:
- LOCATION_VERIFICATION.AREA_NOT_COVERED: The location is not supported
- LOCATION_VERIFICATION.INVALID_AREA: The area is too small, the polygon is too complex, etc 

For codes specific to certain API (not coming from Commonalities) it would be better to prefix them with api name